RIOT-OS/RIOT
# tests/xtimer_mutex_lock_timeout: don't assume number of threads at start

### Contribution description

When CDC ACM is used as stdio the number of threads-in-use at the start of
the test is not always 2, as it used to be.

### Testing procedure

Run `tests/xtimer_mutex_lock_timeout` with `compile_and_test_for_board`. Choose a board which uses CDC ACM as stdio. That means there are three threads active when the test starts. The test should succeed.

Run `tests/xtimer_mutex_lock_timeout` with `compile_and_test_for_board`. Choose a board which uses standard UART for stdio (no CDC ACM). That means there are two threads active when the test starts. The test should succeed.
